README.md

```diff
@@ -60,7 +83,7 @@ You can override these values by setting the environment variables, respectively
 
 `zeppelin/bootstrap/js` and `zeppelin/bootstrap/css` lets you deploy JS and CSS libraries inside Zeppelin.
 
-Call `./zeppelin.sh --refresh` to refresh your container without restarting it!
+__On Linux__, call `./zeppelin.sh --refresh` to refresh your container without restarting it!
```

Comment:
I'm nitpicking here but I would maybe add "On Linux **and macOS**"

Reply:
maybe `Unix-like`?